src/data-access/mutations/useMarkNotificationAsUnseen.ts

```diff
@@ -0,0 +1,36 @@
+import { useMutation } from 'react-query'
+import axios, { AxiosError } from 'axios'
+import { Notification } from 'mockApi/models'
+import { API } from '../API'
+import { queryClient } from '../queryClient'
+import { QueryKeys } from '../QueryKeys'
+
+export type MarkUnseenResponse = { notification: Notification }
+
+const markNotificationAsUnseen = async (id: string): Promise<MarkUnseenResponse> => {
```

Comment:
I think this file can be combined with useMarkNotificationsAsSeen file as it has similar responsibility, just to stay consistent with other hook files